pipeline/load.py

```diff
@@ -0,0 +1,86 @@
+"""Uploads current price data to the RDS database triggered every 3 minutes"""
+import logging
+from psycopg2.extensions import connection
+from dotenv import load_dotenv
+from connect_to_database import get_connection
+
+
+def configure_logging() -> None:
```

Comment:
I've noticed that this function is being repeated in several of your scripts. I think it would be much better if this was imported from another file, probably from `connect_to_database` since you're already importing `get_connection` from there.